### PR TITLE
Feat: Tests added for PR; Closes #1204

### LINKED ISSUE #1204: Tests needed for PR #1198
https://github.com/bytedeck/bytedeck/pull/1198

### PR DESCRIPTION
**Pull request by 9921622**
Please ensure you are familiar with our Pull Request Description expectations described here: https://www.pullrequest.com/blog/writing-a-great-pull-request-description/
### What?
Added two new test cases for ```Tag_get_quest_submission_badge_assertion_by_tag_Tests```
I decided to add these test cases for models because ```DetailView``` explicitly uses ```get_quest_submission_by_tag``` and ```get_badge_assertion_by_tags```

### Why?
See Issue #1204. 

### How?
Added a test to see if only the correct submissions/assertions will show
image
image

### Testing?
See "What"

### Screenshots (if front end is affected)
None

### Anything Else?
None

### Review request
@tylerecouture

### CODE REVIEW

File: src/tags/tests/test_models.py [hunk 1]
```diff
@@ -104,6 +104,61 @@ def test_multiple_badge_assertion_tagged(self):
         badge_assertion_qs = get_badge_assertion_by_tags(self.user, ['tag0', 'tag1', 'tag2'])
         self.assertEqual(badge_assertion_qs.count(), 6)
 
+    def test_correct_quest_submission_tagged(self):
+        """ check if 'correct tag' is retrieved and no other submission from a different tag or no tag """
+        # with correct tag
+        co_1, cs_1 = self.create_quest_and_submissions(1)
```
Comment (tylerecouture) [inline src/tags/tests/test_models.py:110]: don't use variables names like this.  use informative full words, sentences if you have to.  Reading this code, I don't know what data those variables are holding.  what's a co and a ca?

File: src/tags/tests/test_models.py [hunk 2]
```diff
@@ -104,6 +104,61 @@ def test_multiple_badge_assertion_tagged(self):
         badge_assertion_qs = get_badge_assertion_by_tags(self.user, ['tag0', 'tag1', 'tag2'])
         self.assertEqual(badge_assertion_qs.count(), 6)
 
+    def test_correct_quest_submission_tagged(self):
```
Comment (tylerecouture) [inline src/tags/tests/test_models.py:107]: Please re-read CONTRIBUTING.md fully.

Then see test naming convention here: https://github.com/bytedeck/bytedeck/blob/develop/CONTRIBUTING.md#test-requirements

File: src/tags/tests/test_models.py [hunk 3]
```diff
@@ -104,6 +104,61 @@ def test_multiple_badge_assertion_tagged(self):
...
+
+        # check the amount of submissions are correct
+        quest_submission_qs = get_quest_submission_by_tag(self.user, ['correct tag'])
+        self.assertEqual(quest_submission_qs.count(), 2)
+
+        # check if the submissions are the correct ones
+        self.assertTrue(cs_1 in quest_submission_qs)
+        self.assertTrue(cs_2 in quest_submission_qs)
+
+    def test_correct_badge_assertion_tagged(self):
```
Comment (tylerecouture) [inline src/tags/tests/test_models.py:134]: Then see test naming convention here: https://github.com/bytedeck/bytedeck/blob/develop/CONTRIBUTING.md#test-requirements

**Comment by 9921622** [comment]
@tylerecouture review

**Review by tylerecouture — CHANGES_REQUESTED** [review]

File: src/tags/tests/test_models.py (hunk 1 above)
Comment (9921622) [reply src/tags/tests/test_models.py]: fixed it

**Pull request closed without merging**